### PR TITLE
Fix: fixup filters hide hidden files and __pycache__; print shows in …

### PR DESCRIPTION
**Pull request by zack-vii**
…mdsip, err does not

### CODE REVIEW

**Comment by zack-vii** [comment]
This the last time I attempt to fix mitdevices. it just systax correction mostly tabs, print, and Exception, one wrong import

Now that the findPydevices if fixed we see how broken mitdevices are

**Comment by zack-vii** [comment]
can you review this so we can rebase #2066 onto this fix so we can identify potential issues with the changes

File: pydevices/HtsDevices/cryocon18i.py [hunk 1]
```diff
@@ -241,30 +236,29 @@ def store(self):
...
             temps    = trend_temp.data()
             resists  = trend_resis.data()
 
             start_time = times[0]
             for j in range(len(times)):
                 times[j] -= start_time
-                times[j] = float(times[j]) / 1000.    
+                times[j] = float(times[j]) / 1000.
+
+            shot_temp     = self.__getattr__('input_%c' % (i,))
```
Comment (joshStillerman) [inline pydevices/HtsDevices/cryocon18i.py:257]: Why use format above but not here ?  I don't understand the  tradeoffs between various format syntaxes

File: pydevices/HtsDevices/cryocon24c.py [hunk 2]
```diff
@@ -29,31 +27,27 @@ class CRYOCON24C(MDSplus.Device):
...
+    for i in inputs.upper():
+        parts.extend([
+            {'path': ':INPUT_%c' % (i,), 'type': 'signal', 'options': ('no_write_model', 'write_once',)},
+            {'path': ':INPUT_%c:SERIAL_NO' % (i,), 'type': 'TEXT', 'options': ('no_write_shot')},
+            {'path': ':INPUT_%c:CALIBRATION' % (i,), 'type': 'TEXT', 'options': ('no_write_model', 'write_once',)},
+            {'path': ':INPUT_%c:RESISTENCE' % (i,), 'type': 'signal', 'options': ('no_write_model', 'write_once',)},
+        ])
+    for i in loops:
+        parts.extend([
+            {'path': ':LOOP_%d' % (i,), 'type': 'signal', 'options': ('no_write_model', 'write_once',)},
```
Comment (joshStillerman) [inline pydevices/HtsDevices/cryocon24c.py:48]: Same question as above

File: pydevices/MitDevices/dt132.py [hunk 3]
```diff
@@ -40,7 +40,7 @@ class DT132(Device):
     Init(arg) - initialize the DTAO32 device
                 write setup parameters and waveforms to the device
     Store(arg) - store the data acquired by the device
-    Help(arg) - Print this message
+    Help(arg) - print(this message
```
Comment (joshStillerman) [inline pydevices/MitDevices/dt132.py]: extra '('

File: pydevices/acq400Devices/acq400_base.py [hunk 4]
```diff
@@ -81,18 +81,18 @@ def init(self):
 
         # The default case is to use the trigger set by sync_role.
         if self.trig_mode.data() == 'role_default':
-            uut.s0.sync_role = "{} {}".format(self.role.data(), self.freq.data())
+            uut.s0.sync_role = "%s %s" % (self.role.data(), self.freq.data())
```
Comment (joshStillerman) [inline pydevices/acq400Devices/acq400_base.py:84]: same questions about .format

File: pydevices/HtsDevices/cryocon18i.py [hunk 5]
```diff
@@ -82,12 +77,12 @@ def debugging(self):
 
 class CRYOCON18I_TREND(CRYOCON18I):
```
Comment (santorofer) [inline pydevices/HtsDevices/cryocon18i.py:79]: Timo, I think we should leave this section unchanged. I remember that if we didn't do a shallow copy of the CRYOCON18I.parts it wouldn't work correctly. What will be the main reason for this purposed change?
Thanks.

File: pydevices/HtsDevices/cryocon24c.py [hunk 6]
```diff
@@ -303,15 +301,15 @@ def stop(self):
     STOP = stop
 
 class CRYOCON24C_SHOT(CRYOCON24C):
```
Comment (santorofer) [inline pydevices/HtsDevices/cryocon24c.py:303]: The same comment as in CRYOCON18I. I think we should leave the shallow copy in place.
Cheers.

File: pydevices/acq400Devices/acq400_base.py (hunk 4 above)
Comment (zack-vii) [reply pydevices/acq400Devices/acq400_base.py:84]: .format does not work on rhel6.. i.e. old python2.. % is the most compatible variant

File: pydevices/HtsDevices/cryocon18i.py (hunk 5 above)
Comment (zack-vii) [reply pydevices/HtsDevices/cryocon18i.py:79]: the + [ does the copy for you

File: pydevices/HtsDevices/cryocon24c.py (hunk 6 above)
Comment (zack-vii) [reply pydevices/HtsDevices/cryocon24c.py:303]: same here list + list makes a copy

File: pydevices/HtsDevices/cryocon18i.py (hunk 1 above)
Comment (zack-vii) [reply pydevices/HtsDevices/cryocon18i.py:257]: never use .format

File: pydevices/HtsDevices/cryocon24c.py (hunk 2 above)
Comment (zack-vii) [reply pydevices/HtsDevices/cryocon24c.py:48]: never use .format but use % for compatibility

File: pydevices/HtsDevices/cryocon18i.py (hunk 1 above)
Comment (zack-vii) [reply pydevices/HtsDevices/cryocon18i.py:257]: real issue was the c instead of i

File: pydevices/HtsDevices/cryocon18i.py (hunk 5 above)
Comment (zack-vii) [reply pydevices/HtsDevices/cryocon18i.py:79]: extra use of copy package .. unnecessary. list(parts would accomplish the same) and in this case using list + list is better for allocation

File: pydevices/HtsDevices/cryocon24c.py (hunk 6 above)
Comment (zack-vii) [reply pydevices/HtsDevices/cryocon24c.py:303]: and you are right.. you have to make a copy if you intend to append or extend as they reuse the object and would add to the base_parts as well

Comment (santorofer) [reply pydevices/HtsDevices/cryocon24c.py:303]: Ok, I understand. Then it should be fine. Thank you.

**Pull request merged**